CRM/Event/Form/Task/PDF.php

```diff
@@ -99,6 +96,21 @@ public function setDefaultValues() {
    */
   public function listTokens() {
     $tokens = CRM_Core_SelectValues::contactTokens();
+    $tokens = array_merge(CRM_Core_SelectValues::eventTokens(), $tokens);
+    // unset contact_email and contact_phone tokens.
+    // These are location_email and location_contact
+    // Should be cleaned up after ActionSchedule token replacement cleanup.
+    unset($tokens['{event.contact_email}']);
+    unset($tokens['{event.contact_phone}']);
+    $customEventTokens = CRM_CORE_BAO_CustomField::getFields('Event');
```

Comment:
Pretty sure it needs to be `CRM_Core_BAO_CustomField` (i.e. `Core` in mixed case).